src/less/mixins/pagination/ds6/pagination-mixins.less

```diff
@@ -0,0 +1,52 @@
+@import "../../../variables/ds6/color-variables.less";
+@import "../../../variables/ds6/icon-variables.less";
+@import "../../../variables/ds6/typography-variables.less";
+@import "../base/pagination-mixins.less";
+
+.pagination-next-icon() {
```

Comment:
I agree with Mike about a lot of mixins, maybe we can have a state mixin or something. 

Something like 
```
.pagination-disabled(arrow) {
   .background-icon-image-mixin('arrow-@{arrow}-disabled');
}
```
-OR-
```
.pagination-icon(direction, type) {
     // If statement for checking if type is there and then do not add it to the end
    .background-icon-image-mixin('arrow-@{arrow}-@{type}', 17px, 17px); 
}

// And then have one for light without the 17px
```

And the use it like `.pagination-icon('left', 'disabled');` Or whatnot. So there's less repetition.

Reply:
Sure. Let me take a look. At the moment it all works, but I could make it a little less verbose.